hvplot/util.py

```diff
@@ -741,6 +753,10 @@ def import_geoviews():
     return geoviews
 
 
+def _is_installed(package: str) -> bool:
```

Comment:
Maybe cache this result?

Reply:
Yes thanks good catch,  I didn't know `find_spec` was looking at modules. I removed `_is_installed` entirely as I think it was a bit misleading and isn't used much anyway.

Reply:
> Maybe cache this result?

I thought about it before, and then thought people do special things in their environment while working directly from a notebook (install packages, autoreloading modules, etc.). The function that patches plotting signatures and docstrings doesn't only run on import but also when switching HoloViews backend. So I'd rather not cache for now, and keep that for future improvements to improve the import speed of hvPlot, which I'm sure we'll look at at some point.